unstable/lens/isomorphism.scrbl

```diff
@@ -46,6 +46,16 @@ example, are defined like this:
     (make-isomorphism-lenses string->symbol symbol->string))
 ]}
 
+@defproc[(isomorphism-compose [lens isomorphism-lens?] ...) isomorphism-lens?]{
+Like @racket[lens-compose], but works only on isomorphism lenses, and returns an
+isomorphism lens.
```

Comment:
This should mention that it's more efficient in terms of creating fewer wrapping functions than `lens-compose`, otherwise it's not immediately apparent there's ever a reason to use this over `lens-compose`.